backend/danswer/connectors/web/connector.py

```diff
@@ -58,6 +58,8 @@ def __init__(
     ) -> None:
         if "://" not in base_url:
             base_url = "https://" + base_url
+        if not base_url.endswith("/"):
```

Comment:
Might want to check out main, we had this one removed and addressed differently. The issue with this approach is that if you have someone trying to index some page.html doc.pdf etc it will break.

Reply:
Ah I see. In case you start with some index page? Make sense. I will remove it.

Reply:
I just tried that case `http://localhost/index.html` will not work for anything except that one page. Links to the same level `http://localhost/about.html` or even subfolders such as `http://localhost/imprint/index.html` are ignored. Is that expected?

Reply:
Ya it should not include anything that's not under the base path provided. If someone specifies http://localhost/index.html, it should only index the index page, similarly if they add http://localhost/about.html, it should only index the about page.

Reply:
It's better to remove this, for example if someone wants to index https://arxiv.org/pdf/2305.09600.pdf, it should work